pkg/autoops/api/api.go

```diff
@@ -859,23 +863,36 @@ func (s *AutoOpsService) ExecuteAutoOps(
 		if err != nil {
 			return err
 		}
-		if autoOpsRule.AlreadyTriggered() {
-			return errAlreadyTriggered
-		}
 		handler := command.NewAutoOpsCommandHandler(editor, autoOpsRule, s.publisher, req.EnvironmentNamespace)
 		if err := handler.Handle(ctx, req.ChangeAutoOpsRuleTriggeredAtCommand); err != nil {
 			return err
 		}
 		if err = autoOpsRuleStorage.UpdateAutoOpsRule(ctx, autoOpsRule, req.EnvironmentNamespace); err != nil {
+			if err == v2as.ErrAutoOpsRuleUnexpectedAffectedRows {
```

Comment:
If there are no affected rows, it will not treat as an error. Otherwise, it might trigger the alert.